internal/infrastructure/kubernetes/proxy/resource_provider_test.go

```diff
@@ -746,3 +746,55 @@ func loadHPA(caseName string) (*autoscalingv2.HorizontalPodAutoscaler, error) {
 	_ = yaml.Unmarshal(hpaYAML, hpa)
 	return hpa, nil
 }
+
+func TestLackOwnerGatewayLabels(t *testing.T) {
```

Comment:
```suggestion
func TestOwningGatewayLabelsAbsent((t *testing.T) {
```
Please also change it here

Reply:
This was an oversight on my part, I have corrected it